assets/src/design-system/components/search/input/components.js

```diff
@@ -37,8 +37,8 @@ export const InputContainer = styled.div(
     box-sizing: border-box;
     height: 36px;
     width: 100%;
-    max-width: 276px;
-    background-color: ${theme.colors.bg.primary};
+    max-width: 400px;
+    background-color: transparent;
```

Comment:
@BrittanyIRL, both of these were needed to implement this, and I'm not sure if I'm breaking something else by doing this. Can you please double-check other usages of this component, thanks?

Reply:
This will be fine - heads up i have this: https://github.com/google/web-stories-wp/pull/6434/files#diff-2730ae2b9bb49a1f43c5ca0a2f5c89a655ceb46a0b707e0514843e633567a6af waiting for QA right now that will fix the input to always be left aligned. And there's a fix for the menu type sizing in another PR waiting for QA.

Reply:
No problem, that'll just be rolled out then.